src/components/Link.js

```diff
@@ -26,7 +26,7 @@ class CSVLink extends React.Component {
 
   componentDidUpdate(prevProps) {
     if (this.props !== prevProps) {
-      const { data, headers, separator, uFEFF } = prevProps;
+      const { data, headers, separator, uFEFF } = this.props;
```

Comment:
@abdennour This seems like a P0 bug, it breaks the library altogether.

Reply:
Thank you guys.. for next issues, please contact me on linkedin http://in.abdennoor.com